src/main/java/komo/fraczek/toukparking/resource/DriverController.java

```diff
@@ -0,0 +1,46 @@
+package komo.fraczek.toukparking.resource;
+
+import komo.fraczek.toukparking.domain.DriverType;
+import komo.fraczek.toukparking.domain.ParkingBill;
+import komo.fraczek.toukparking.service.ParkingService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class DriverController {
+
+    private static final Logger logger = LoggerFactory.getLogger(DriverController.class);
+
+    ParkingService parkingService;
+
+    @Autowired
+    public void setParkingService(ParkingService parkingService) {
+        this.parkingService = parkingService;
+    }
+
+    //    there is no need of passing driver type, REGULAR is default
+    @PostMapping(path = {"/start_parking_meter/{numberPlate}/{driverType}", "/start_parking_meter/{numberPlate}" })
```

Comment:
czy opcjonalnych parametrów nie da się zamodelować lepiej niż w ścieżce?

Reply:
To ja trochę w innym temacie: nie wiedziałem, że da się w ten sposób zmapować ścieżkę na metodę, dzięki za pokazanie mi tego :)